operator/config/webhook/manifests.yaml

```diff
@@ -27,45 +27,3 @@ webhooks:
     resources:
     - seldondeployments
   sideEffects: None
-- admissionReviewVersions:
```

Comment:
Seems this got included again - to be honest I think it's overdue to add this change, I remember we did discuss to add this on a separate PR, but we could just add it as otherwise it may keep coming up